packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/prepare-api/index.md

```diff
@@ -108,12 +108,12 @@ A best practice is to use a generated globally unique identifier (GUID) for this
 
 Okta user management requires that your SCIM API supports an `active` attribute for each user resource that can be set to `true` or `false` to denote a resource as "active" or "inactive".
 
-## SCIM Facade
+## SCIM facade
```

Comment:
Are the different header levels on this page intentional? Most are H3, but this one is an H2.

Reply:
I have added a new H2 "Features" to separate the page properly. Thanks for the idea!